### DISCUSSION TITLE
How can we remove channels or add them to the https m3u link.

### DISCUSSION THREAD

**Discussion by harmojim76** [iptv-org/community#495] [Q&A]
Hello, 

how can we remove channels or add them to the https m3u link. I know how to do it with notepad but then to put it back in the http link what can we do?

I have other iptv channels that are not in your list and I would like to add them. I know how to do it with Notepad and using Smart IPTV on Android I can add the m3u file. But with this same file I don't know how to convert it to put it online http afterwards. Can you help me ?

Thx for all

Chris

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
